notebook/tests/selenium/utils.py

```diff
@@ -184,6 +184,11 @@ def add_cell(self, index=-1, cell_type="code", content=""):
         if cell_type != 'code':
             self.convert_cell_type(index=new_index, cell_type=cell_type)
 
+    def delete_cell(self, index):
+        self.focus_cell(index)
+        self.to_command_mode
```

Comment:
I think this should have `()` at the end, otherwise it's not doing anything. I can see it's just moved from another file, but it's worth checking while we're looking at it.

Reply:
my bad, I should have noticed. I will check on this.